README.md

```diff
@@ -361,7 +361,22 @@ and if it finds any builds that are newer than the most recent deployment, it wi
 1.  Click "Create pull request."
 1.  When the pull request is approved, merge it. This will trigger a build that will automatically deploy `master` to production.
 
-### Contributing
+### Deploying Manually
+
+In the unusual case that Train or Production have not been automatically deployed (which should happen within five minutes of a finished Lint and Build action on the appropriate branch), it is possible to deploy manually. 
+
+1. Clone/open the repo in VSCode, check out the branch you want to deploy, which should be `develop` for Train and `master` for Production, and fetch and pull the most uptodate commit(s). 
+1. Build the project by running `npm run build` in VSCode on a clone of the project set to the branch you want to deploy. The output will be in `path/to/the/repo/gordon-360-ui/build`.
+1. Connect to the `360-Frontend.gordon.edu` server. See [RemoteDesktopToVM](https://github.com/gordon-cs/gordon-360-api/blob/develop/RemoteDesktopToVM.md) in the API repo for instructions on how to connect.
+1. Open File Explorer and navigate to `D:\wwwroot\.
```

Comment:
```suggestion
1. Open File Explorer and navigate to `D:\wwwroot\`.
```

I think you forgot a backtick there

Reply:
Weird that I didn't notice that before. Thanks, I'll add it to the next documentation PR for develop. Since this is a release though it won't matter because the develop branch is what displays on github and master will get updated with the next release but no one reads the master version of the README